lib/components/Table.tsx

```diff
@@ -232,7 +231,6 @@ export const Table = <T, >({
   return (
     <div className={tw('flex flex-col gap-y-4 overflow-hidden')}>
       <div>
-        <SimpleBarReact>
```

Comment:
Why did you remove the Scrollbar wrapper and not replace it? Does the new scrollbar appear automatically?

Reply:
Not sure if we need it really because we have the pagination, so there should be the need of scrolling. The Problem is react-custom-scrollbars-2 come with some side effects, on solution I found is to remove it or to use simplebar

Reply:
[Uploading Bildschirmaufzeichnung vom 2023-09-02, 12-51-29.webm…]()